Feature/coding exercise lottery game (#1)
Geethanjali-Nagendra/coding_exercises: README.md

# coding_exercises

> This Project contains a simple Python programs, I have written to test my understanding of Python concepts through hands-on practice.

Reviewer: never use "I" , "me" in public repositories
```suggestion
This Project contains a simple Python programs which explains understanding of Python concepts through hands-on practice.
```
